iron-autogrow-textarea.html

```diff
@@ -324,6 +309,7 @@
         textarea.value = !(this.bindValue || this.bindValue === 0) ? '' : this.bindValue;
       }
 
+      this.value = this.bindValue;
```

Comment:
it should be possible to move this line at the beginning of this method, and delete `ready`